README.md

```diff
@@ -299,14 +299,25 @@ Kubernetes Engine][connect-to-k8s].
 
 ## Running behind a Socks5 proxy
 
-The Cloud SQL Auth Proxy includes support for sending requests through a SOCKs5
-proxy. If a Socks5 proxy is running on `localhost:8000`, the command to start
+The Cloud SQL Auth Proxy includes support for sending requests through a SOCKS5
+proxy. If a SOCKS5 proxy is running on `localhost:8000`, the command to start
 the Cloud SQL Auth Proxy would look like:
 
 ```
-ALL_PROXY=socks5://localhost:8000 cloud_sql_proxy -instances=$INSTANCE_CONNECTION_NAME=tcp:5432
+ALL_PROXY=socks5://localhost:8000 \
+HTTPS_PROXY=socks5://localhost:8000 \
+    cloud_sql_proxy -instances=$INSTANCE_CONNECTION_NAME=tcp:5432
 ```
 
+Note: the `ALL_PROXY` environment variable specifies the proxy for all TCP
+traffic to and from a Cloud SQL instance. The `ALL_PROXY` environment variable
+supports `socks5` and `socks5h` protocols. To route DNS lookups through a proxy,
+use the `socks5h` protocol.
+
+The `HTTPS_PROXY` (or `HTTP_PROXY`) specifies the proxy for all HTTP(S) traffic. See
```

Comment:
Can we specify that `HTTPS_PROXY` is for interactions the SQLAdmin API? They also do not need to use HTTPS unless they specifically want their API calls to go through the socks5 proxy

Reply:
Done.